src/main/kotlin/com/hibob/academy/employee_feedback_feature/resource/FeedbackResource.kt

```diff
@@ -53,7 +58,58 @@ class FeedbackResource(private val feedbackService: FeedbackService) {
         return if (hasPermission(employeeRole, Permissions.VIEW_ALL_FEEDBACKS)) {
             Response.ok(feedbackService.getAllFeedbacks(companyId)).build()
         } else {
-            Response.status(Response.Status.FORBIDDEN).entity("UNAUTHORIZED: You do not have access to view feedbacks").build()
+            Response.status(Response.Status.UNAUTHORIZED)
```

Comment:
same comment about the if permission (like in the other pr).

you can do - if (no permission) return
rest of the code not in if block

Reply:
Fixed!